docs/migrations.rst

```diff
@@ -413,6 +413,8 @@ Column types are specified as strings and can be one of:
 -  date
 -  binary
 -  boolean
+-  enum
```

Comment:
Like the Postgres note, I think these two should go in a separate MySQL section.